draft-ietf-core-new-block.xml

```diff
@@ -901,15 +885,18 @@ missing-block-number = uint
         target="RFC7252"></xref>).</t>
 
         <t>NON_RECEIVE_TIMEOUT is the maximum time to wait for a missing
-        payload before requesting retransmission. NON_RECEIVE_TIMEOUT has a
-        value of twice NON_TIMEOUT.</t>
+        payload before requesting retransmission. This is the initial timeout
+        for the first request of a missing payload. Every time the missing
+        payload is re-requested, the timeout value doubles. NON_RECEIVE_TIMEOUT
+        has a default value of twice NON_TIMEOUT. NON_RECEIVE_TIMEOUT MUST
+        always be greater than NON_TIMEOUT by at least one second.</t>
```

Comment:
```suggestion

```

Reply:
I wonder whether some text can be added to explain the rationale for the one second mentioned in the text.

Reply:
Text added